kore/src/Kore/Repl/State.hs

```diff
@@ -654,13 +654,13 @@ substituteAlias
         SimpleArgument str -> str
         QuotedArgument str -> "\"" <> str <> "\""
 
-createOnePathClaim
+createClaim
     :: Claim claim
     => (claim, TermLike Variable)
-    -> Rule.OnePathRule Variable
-createOnePathClaim (claim, cpattern) =
-    Rule.OnePathRule
-    $ Rule.RulePattern
+    -> claim
+createClaim (claim, cpattern) =
+    fromRulePattern claim
+    Rule.RulePattern
```

Comment:
The formatting here looks a bit off since it's not indented compared to the previous line. It feels like a do block.